app/views/pages/dfe_signin_help.html.erb

```diff
@@ -21,8 +21,7 @@
     </div>
 
     <p>
-      Don't worry if you don't have or don't know your details - all you need to
-      do is contact your organisation's DfE Sign-in approver.
+      If you do not have or have forgotten your details, contact your organisation's DfE Sign-in approver.
```

Comment:
@ekumachidi please can you add a comma after the 1st have - If you do not have, or have forgotten...